.github/check_metaschema_compatibility.py

```diff
@@ -0,0 +1,40 @@
+import glob
+import re
+import semver
+import subprocess
+
+metaschema_file_version_pattern = "^metaschema/near-abi-(\d+\.\d+\.\d+)-schema\.json$"
+rust_schema_version_pattern = "^pub const SCHEMA_VERSION: &str = \"(\d+\.\d+\.\d+)\";$"
+
+persisted_semvers = []
+for metaschema in glob.glob("metaschema/near-abi-*.*.*-schema.json"):
+    persisted_semver = semver.VersionInfo.parse(
+        re.match(metaschema_file_version_pattern, metaschema).group(1)
+    )
+    persisted_semvers.append(persisted_semver)
+persisted_semvers.sort()
+last_persisted_semver = persisted_semvers[-1]
+print("Last persisted ABI schema version:", last_persisted_semver)
+
+with open("near-abi/src/lib.rs", "r") as sources:
+    for line in sources.readlines():
+        match = re.match(rust_schema_version_pattern, line)
+        if match is not None:
+            current_semver = semver.VersionInfo.parse(match.group(1))
+            break
+print("Current ABI schema version:", current_semver)
```

Comment:
Just in case a PR removes this line, or modifies it in any way, we should probably report an error. Because we were unable to extract the version.